docs/source/introduction.rst

```diff
@@ -1,23 +1,36 @@
 Introduction to Glow
 ====================
 
-Glow aims to simplify genomic workflows at scale. The best way to accomplish this goal is to take a system that
-has already been proven to work and adapt it to fit into the genomics ecosystem.
+Genomics data has been doubling every seven months globally. It has now reached a scale where genomics has 
```

Comment:
Should it be genomic data or genomics data?

Reply:
I've seen it used both ways